test/add-member.test.js

```diff
@@ -25,64 +26,42 @@ test('get added to a group', async (t) => {
     ),
   })
 
-  await Promise.all([alice.tribes2.start(), bob.tribes2.start()])
-  t.pass('tribes2 started for both alice and bob')
+  await run(
+    'tribes2 started for both alice and bob',
+    Promise.all([alice.tribes2.start(), bob.tribes2.start()])
+  )
 
   await p(alice.metafeeds.findOrCreate)()
   const bobRoot = await p(bob.metafeeds.findOrCreate)()
 
-  await replicate(alice, bob)
-  t.pass('alice and bob replicate their trees')
+  await run('alice and bob replicate their trees', replicate(alice, bob))
```

Comment:
I added this run function because it tidies up the tests and also gives much better logging for free.
relevant because I was debugging some replicate errors